src/orion/benchmark/task/profet/xgboost.py

```diff
@@ -0,0 +1,74 @@
+""" Simulated Task consisting in training an Extreme-Gradient Boosting (XGBoost) predictor.
+
+The task is created using the Profet algorithm:
+
+```
+@misc{klein2019metasurrogate,
```

Comment:
Seing this again and again, I'm thinking maybe it should only be in the main module. :)

Reply:
Moved the references (with the udated style) to the __init__.py